pgmq-extension/sql/pgmq.sql

```diff
@@ -571,11 +571,11 @@ BEGIN
     atable
   );
 
-  IF NOT pgmq._belongs_to_pgmq(FORMAT('%I', qtable)) THEN
+  IF NOT pgmq._belongs_to_pgmq(qtable) THEN
```

Comment:
Okay, from the code:

https://github.com/tembo-io/pgmq/blob/78a766bd750b5985ac590ac384b8d294601fe544/pgmq-extension/sql/pgmq--1.1.1--1.2.0.sql#L166-L170

It looks like `_belongs_to_pgmq()` queries `pg_class`. For that case, you don't want them quoted as identifiers, but you _do_ want them to be formatted as expected. I think this is the correct approach, since we lowercase everything, but any special characters will be preserved.

IOW, if `qtable` is `q_hello there`, that's exactly how it will be read from `pg_class`, as — long as we _consistently_ lowercase entity names.